pyproject.toml

```diff
@@ -54,6 +54,7 @@ ignore = [
   "PT011",
   "PT012",
   "PT013",
+  "PT015",
```

Comment:
I'd prefer not to ignore this; if it's statically known we should `raise AssertionError`.

Reply:
agreed, but "don't use `assert false`" is already covered by another B011, which is still enabled here.  (see [CI job](https://github.com/HypothesisWorks/hypothesis/actions/runs/15174352515/job/42671573113) where both were enabled). I just disagree with "use pytest.fail".